doc/index.rst

```diff
@@ -28,6 +28,18 @@ specify along with the Jupyter Notebooks inside. You will be able to share a URL
 with users that can immediately begin interacting with this environment via the
 cloud.
 
+What resources are given to Binder users?
+-----------------------------------------
+A Binder user is somebody that clicks on a Binder link in order to run a
+repository that someone has prepared. The public ``mybinder.org`` deployment
+gives users a minimum of 1G and a maximum of 4G of RAM. Custom BinderHub
```

Comment:
Strike all information about Custom BinderHub / settting up your own BinderHub in this section since these docs focus on the mybinder deployment.

Instead, add a final question:

**What can I do if mybinder does not meet my needs?**

Follow with the how to deploy your own BinderHub.

Reply:
To expand on 'minimum' and 'maximum':

1G is the 'request' - so all spawned user notebooks will get *at least* 1G of RAM. If the cluster is not big enough to give them 1G of RAM, spawning will just fail instead. Pods using under 1G of RAM will also never be killed for out of memory reasons.

4G is the 'limit' - depending on conditions on each machine, each notebook can use upto 4G of RAM. If it tries to use more than 4G of RAM it will *definitely* fail. Between 1G and 4G is a grey area of 'best effort' - your notebook might or might not be killed depending on local conditions.

I feel we should communicate this very clearly, and 'minimum' & 'maximum' misses the nuances. Perhaps the way to do this is to discuss this somewhere else and link to it from here? Am unsure what the prose might look like.